gloo/test/multiproc_test.h

```diff
@@ -15,23 +15,34 @@
 #include <chrono>
 #include <string>
 
+#include "base_test.h"
 #include "gloo/common/logging.h"
 #include "gloo/rendezvous/context.h"
 #include "gloo/rendezvous/file_store.h"
-#include "gloo/transport/tcp/device.h"
+#include "gloo/test/base_test.h"
+#include "gloo/test/openssl_utils.h"
+#include "gloo/transport/tcp/tls/device.h"
 
 namespace gloo {
 namespace test {
 
 const int kExitWithIoException = 10;
 const auto kMultiProcTimeout = std::chrono::milliseconds(500);
 
+const std::vector<Transport> kTransportsForMultiProcTest {
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
+    Transport::TCP_TLS,
+#endif
+    Transport::TCP,
+};
+
 class MultiProcTest : public ::testing::Test {
  protected:
   void SetUp() override;
   void TearDown() override;
 
-  void spawnAsync(int size, std::function<void(std::shared_ptr<Context>)> fn);
+  void spawnAsync(Transport transport, int size,
```

Comment:
is this what clang-format gives? shouldn't it be either one arg per-line or all args in one line?

Reply:
my clang-format gives this